students/piechowski.michal/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+list = [int(x) for x in input().split()]
```

Comment:
nie uzywaj `list` jako nazwa zmiennej, ponieważ jest to równiez nazwa funkcji `list()`